### PR TITLE
Update core

### PR DESCRIPTION
**Pull request by digeff**
Update to new core so we won't call RS5 APIs in RS4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "portscanner": "^2.2.0",
-    "vscode-chrome-debug-core": "^6.7.8",
+    "vscode-chrome-debug-core": "^6.7.12",
     "vscode-debugadapter": "^1.31.0-pre.0",
```
Comment (mrcrane) [inline package.json]: Should we update vscode-debugadapter to "1.31.0" to match -core?

Comment (digeff) [reply package.json]: Yes. Thanks for catching that :)

File: package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
   "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "portscanner": "^2.2.0",
-    "vscode-chrome-debug-core": "^6.7.8",
+    "vscode-chrome-debug-core": "^6.7.12",
```
Comment (dhanvikapila) [inline package.json]: I just added a comment on the merge you did on core.. do take a look.

Comment (digeff) [reply package.json]: It can be NaN only and it'll behave as we want it to in that case... Good catch! :)

**Pull request merged**